cmd/node/main.go

```diff
@@ -669,15 +676,21 @@ func startNode(ctx *cli.Context, log *logger.Logger, version string) error {
 		networkComponents,
 		uint64(ctx.GlobalUint(bootstrapRoundIndex.Name)),
 		version,
+		elasticIndexer,
 	)
 	if err != nil {
 		return err
 	}
 
+    if shardCoordinator.SelfId() == sharding.MetachainShardId {
+        indexValidatorsListIfNeeded(elasticIndexer, nodesCoordinator)
```

Comment:
run go fmt or check the settings of your IDE you tabs vs spaces. Please check also the rest of the files in the commit

Reply:
Done. Now i m using tabs.